contracts/UniswapV3PriceFeed.sol

```diff
@@ -17,42 +17,37 @@ contract UniswapV3PriceFeed is IUniswapV3PriceFeed, BlockContext {
     // STATE
     //
 
-    address public pool;
-
     uint32 internal constant _TWAP_INTERVAL = 30 * 60;
 
+    address public pool;
+
     //
-    // EXTERNAL NON-VIEW
+    // EXTERNAL
     //
 
     constructor(address poolArg) {
-        // EPF_EANC: pool address is not contract
-        require(address(poolArg).isContract(), "EPF_EANC");
+        // UPF_PANC: pool address is not contract
+        require(address(poolArg).isContract(), "UPF_PANC");
 
         pool = poolArg;
     }
 
-    //
-    // EXTERNAL VIEW
-    //
-
     function getPrice() external view override returns (uint256) {
         uint256 markTwapX96 = _formatSqrtPriceX96ToPriceX96(_getSqrtMarkTwapX96(_TWAP_INTERVAL));
         return _formatX96ToX10_18(markTwapX96);
     }
 
-    //
-    // EXTERNAL PURE
-    //
-
     function decimals() external pure override returns (uint8) {
         return 18;
     }
 
-    /// @dev if twapInterval < 10 (should be less than 1 block), return mark price without twap directly,
+    //
+    // INTERNAL
+    //
+
+    /// @dev if twapInterval < 10 (should be less than 1 block), return mark price without twap directly
```

Comment:
Let's call it market price. L52 and L53, should be somewhat like sqrtMarketPrice